chapters/ch07.asciidoc

```diff
@@ -1480,7 +1480,7 @@ const items = [1, 2, 3, ,,,,,,,]
 // <- [1, 2, 3, undefined x 7]
 ----
 
-The function call shown below takes the +items+ array and determines that it'll start "pasting" items in the sixth position. It further determines that the items to be copied will be taken starting in the second position (zero-based), until the third position (also zero-based).
+The function call shown below takes the +items+ array and determines that it'll start "pasting" items in the sixth position (zero-based). It further determines that the items to be copied will be taken starting in the second position (zero-based), until the third position (also zero-based, but excluded).
```

Comment:
Let's remove the second and third mentions of `zero-based`. In the last bit we could probably get away with `third position (not inclusive)`